profiles/test.yaml

```diff
@@ -0,0 +1,318 @@
+check_unique_id:
```

Comment:
I think you committed this by mistake. Currently, there is no profile stored in code.

Reply:
As discussed in last demo, a sample profile is required to be stored. I assumed it to be stored alongside code.

Reply:
will remove it from code.

Reply:
This is still here